repository_service_tuf_api/config.py

```diff
@@ -91,9 +95,14 @@ def put(payload: PutPayload):
         queue="metadata_repository",
         acks_late=True,
     )
-    data = {"task_id": task_id}
 
-    return PutResponse(data=data, message="Settings successfully submitted.")
+    data = {
+        "task_id": task_id,
+        "last_update": datetime.now(),
+    }
+    message = "Settings successfully submitted."
```

Comment:
Try adding the message directly on line 105.
If the line becomes more than 79 characters, then leave it as is.